docs/faq.md

```diff
@@ -12,6 +12,28 @@ hr {
 }
 </style>
 
+<hr id="operation-buffering-timed-out" />
+
+<a class="anchor" href="#operation-buffering-timed-out">**Q**</a>. Operation `...` timed out after 10000 ms. What gives?
+
+**A**. At its core, this issue stems from being unable to connect to mongoose. If you are using an application, like a bot,
+that wishes to connect to mongoose, you must ensure that you are connecting before the bot bot activates. For example, when working
+with discord bots, the last line of your index file is `client.login(token)`. You want to place the connection string, `mongoose.connect().then()` before this line.
+Do not place it in an event handler like `client.on('message')`, as this activates after you have logged in to the bot.
+
+<a class="anchor" href="#not-local"> **Q**</a>. I am able to connect locally but when I try to connect to a server I get this error. What gives?
+
+You must ensure that you have whitelisted your ip on [mongodb](https://docs.atlas.mongodb.com/security/ip-access-list/) to let it connect.
+If security is of no concern for your application, like a personal project that doesn't use user's information, you can allow access from all
```

Comment:
Get rid of the "if security is of no concern" bit, that isn't necessarily true. Just put "You can allow access from all IPs with ..."